nox/_options.py

```diff
@@ -422,7 +475,8 @@ def _tag_completer(
             "--no-reuse-existing-virtualenvs",
         ),
         group=options.groups["environment"],
-        help="Reuse existing virtualenvs instead of recreating them.",
+        help="This is an alias for '--reuse-venv=yes|no'.",
```

Comment:
I'd love to see this called out in the docs as well

Reply:
It should be in the docs, near the end of this statement in `config.rst`:

`* ``nox.options.reuse_existing_virtualenvs`` is equivalent to specifying :ref:`--reuse-existing-virtualenvs <opt-reuse-existing-virtualenvs>`. You can force this off by specifying ``--no-reuse-existing-virtualenvs`` during invocation. Alias of ``nox.options.reuse_venv``.`

Reply:
The docs should state explicitly that `--reuse-venv=yes` is equivalent to `--reuse-existing-virtualenvs`.

I'm happy for this to land as-is though. We probably want to deprecate the longer version anyway.

(OT: It might also be a nice improvement to allow specifying `--reuse-venv` without a value, defaulting to `yes`.)

Reply:
Agree, I didn't explicitly deprecate it on this PR but it can be done so in a follow up one. When that happens I think it makes sense to augment `--reuse-venv` to default to `yes` when specified. Note `-r` is a valid shorthand for `--reuse-venv=yes`

Reply:
> The docs should state explicitly that --reuse-venv=yes is equivalent to --reuse-existing-virtualenvs.

Done in fdf876c16f2605ad4319c0bc42f68108ad15ceb6